src/H5FDpublic.h

```diff
@@ -197,6 +197,7 @@
 #define H5FD_CTL__MEM_ALLOC                   5
 #define H5FD_CTL__MEM_FREE                    6
 #define H5FD_CTL__MEM_COPY                    7
+#define H5FD_CTL__GET_NUM_REVISIONS           8
```

Comment:
We can do this in the short term, but this is going to be a problem going forward. Many VFDs will want to accept values like this, but they won't be able to modify H5FDpublic.h, so we need a system that generalizes to all VFDs.